DomainContent/Trivia/FourSquareTrivia/triviaColorCheckBlue.js

```diff
@@ -0,0 +1,116 @@
+// triviaColorCheck.js
```

Comment:
update file name...same for other files like this one

Reply:
fixed